src/docs/components/Nav.js

```diff
@@ -99,8 +99,9 @@ export default function Nav () {
             {
               label: 'Components',
               items: [
-                { label: 'Box', route: routes.components.breadcrumb },
-                { label: 'Navbar', route: routes.components.navbar }
+                { label: 'Breadcrumb', route: routes.components.breadcrumb },
```

Comment:
I think was a small typo here, changed from Box to Breadcrumb

Reply:
yes it was a typo, thanx